src/tlo/analysis/hsi_events.py

```diff
@@ -47,7 +47,7 @@ def get_hsi_event_classes_per_module(
         module = importlib.import_module(f'tlo.methods.{module_name}')
         tlo_module_classes = [
             obj for _, obj in inspect.getmembers(module)
-            if is_valid_tlo_module_subclass(obj, excluded_modules)
+            if is_valid_tlo_module_subclass(obj, {})
```

Comment:
This change is I noticed the previous implementation was not quite correct as `excluded_modules` here is taken to be a set of Python module names (submodules of `tlo.methods` package) while `is_valid_tlo_module_subclass` takes a set of TLO `Module` subclass names for the `excluded_modules` argument.